cryptography/src/secp256r1/scheme.rs

```diff
@@ -256,6 +301,27 @@ pub struct Signature {
     signature: p256::ecdsa::Signature,
 }
 
+impl Codec for Signature {
+    fn write(&self, writer: &mut impl Writer) {
+        writer.write_fixed(&self.raw);
+    }
+
+    fn read(reader: &mut impl Reader) -> Result<Self, CodecError> {
+        let raw = <[u8; SIGNATURE_LENGTH]>::read(reader)?;
+        let signature = p256::ecdsa::Signature::from_slice(&raw)
```

Comment:
You missed: https://github.com/commonwarexyz/monorepo/blob/f7bf94b9970c40123c111182f4fb6543d9a53283/cryptography/src/secp256r1/scheme.rs#L313-L316

Are you just copying these from `try_from` or doing something different?

Reply:
This would otherwise be caught if we used the existing tests I think.